recipe/conda_build_config.yaml

```diff
@@ -12,9 +12,8 @@ c_stdlib_version:         # [osx and x86_64]
 MACOSX_SDK_VERSION:       # [osx and x86_64]
   - "10.15"               # [osx and x86_64]
 
-# see https://github.com/conda-forge/ray-packages-feedstock/pull/152#issuecomment-2015479106
 libgrpc:
-- '1.65'
+- '1.67'
 
 c_stdlib_version:   # [linux]
   - "2.17"          # [linux]
```

Comment:
All of this can be removed, as it matches the current global pinning.

Reply:
So I should remove entirely the sections from the conda_build_config.yaml? Can we remove the earlier lines as well and delete the file?

Reply:
No, just the lines I marked. The MACOS_SDK is newer than our baseline and presumably still necessary